packages/gamut-labs/src/ListSection/__tests__/ListSection-test.tsx

```diff
@@ -66,6 +66,29 @@ describe('ListSection', () => {
     expect(view.getByText('Show Less')).toBeTruthy();
   });
 
+  it('calls onShowAllOrLessClick with true when the Show All button is clicked', () => {
+    const onShowAllOrLessClick = jest.fn();
+    const { view } = renderListSection({ onShowAllOrLessClick });
+
+    const button = view.getByText('Show All');
+    fireEvent.click(button);
+
+    expect(onShowAllOrLessClick).toHaveBeenCalledWith(true);
+  });
+
+  it('calls onShowAllOrLessClick with false when the Show Less button is clicked', () => {
+    const onShowAllOrLessClick = jest.fn();
+    const { view } = renderListSection({ onShowAllOrLessClick });
+
+    const button = view.getByText('Show All');
+
+    fireEvent.click(button); // Switch 'Show All' to 'Show Less'
+    onShowAllOrLessClick.mockClear();
+    fireEvent.click(button);
+
+    expect(onShowAllOrLessClick).toHaveBeenCalledWith(false);
```

Comment:
Nit: I have never encountered a need to call `mockClear` and I am amused by this streak continuing

```suggestion
    fireEvent.click(button);

    expect(onShowAllOrLessClick).toHaveBeenLastCalledWith(false);
```

Reply:
Hahaha! This was actually the first time I had ever used it as opposed to `mockReset/mockRestore` and I was actually proud of myself for it.  Maybe we should submit a pull request to Jest and just remove it 🙃